go/plugins/googleai/googleai.go

```diff
@@ -128,30 +137,25 @@ func Init(ctx context.Context, g *genkit.Genkit, cfg *Config) (err error) {
 		}
 	}
 
-	opts := append([]option.ClientOption{
-		option.WithAPIKey(apiKey),
-		genai.WithClientInfo("genkit-go", internal.Version),
```

Comment:
This is very important. Please look into and see which header is set with this option and manually set it using the `HTTPOptions.Headers` field.